### ISSUE TITLE
Update bioconda bbmap to 39.03

### ISSUE THREAD

**Issue by tamuanand** [bioconda/bioconda-recipes#44112]
Hi @bgruening 

bbmap has a new version - 39.03 - https://sourceforge.net/projects/bbmap/files/

- however, for whatever reason, bioconda has not picked up this update - https://anaconda.org/bioconda/bbmap

Is there a way to track/automate these?

For some reason @pdiakumis had to do this in a semi-automated way for `mosdepth`
- https://github.com/brentp/mosdepth/issues/212#issuecomment-1752793162
- https://github.com/bioconda/bioconda-recipes/pull/43500

**Comment by corneliusroemer** [comment]
Yes you can write a bot that automates these PRs - autobump might not work for sourcefourge hosted repos.

**Comment by aliciaaevans** [comment]
Autobump does work for SourceForge as long as the URL matches the regex for the [SourceForge hoster](https://bioconda.github.io/contributor/updating.html#writing-a-new-hoster), which bbmap does.  Running autobump for bbmap locally, it looks like the version parser that is used to parse and sort the versions to find the latest is choking on the old versions that contain `_java7`.
